### DISCUSSION TITLE
ARM64 support

### DISCUSSION THREAD

**Discussion by runesoerensen** [heroku/buildpacks#9] [Ideas]
Looking for ARM64 support? We're working on it! Subscribe to this discussion to receive early updates as we roll out multi-arch support across our CNB suite.

If you have questions, feedback, recommendations, ideas, please don't hesitate to share - we want ot hear from you!

**Comment by runesoerensen** [comment]
We recently published a multi-arch [`heroku/builder:24` builder](https://github.com/heroku/cnb-builder-images?tab=readme-ov-file#available-images), which is based on the `heroku-24` base images and ubuntu 24.04.

The new builder is still in active development and currently includes Go, Node.js Python and Procfile buildpacks. It's [available on Docker Hub](https://hub.docker.com/r/heroku/builder/tags) like our other builders and can be used like so:

    pack build my-sample-app --builder heroku/builder:24

**Reply by schneems** [reply]
As of today `heroku/ruby` is in the `heroku/builder:24` and works with ARM on heroku-24 though support for JRuby is in-progress.

**Reply by schneems** [reply]
Jruby work is here https://github.com/heroku/docker-heroku-jruby-builder/pull/33

**Reply by schneems** [reply]
Java merged https://github.com/heroku/cnb-builder-images/pull/526

**Comment by edmorley** [comment]
`heroku/builder:24` now supports all of Heroku's officially supported languages - so ARM64 CNB support is now complete!

See:
https://github.com/heroku/buildpacks/discussions/25
